plugin/pxr/maya/lib/usdMaya/exportCommand.h

```diff
@@ -24,23 +24,19 @@
 
 #include <maya/MPxCommand.h>
 
+#include <mayaUsd/fileio/commands/baseExportCommand.h>
+
 PXR_NAMESPACE_OPEN_SCOPE
 
 
-class UsdMayaExportCommand : public MPxCommand
+class PxrMayaUSDExportCommand : public MayaUsd::MayaUSDExportCommand
 {
```

Comment:
I have left the file name as it was before, but I did change the name of the c++ class to include "Pxr" as a way to distinguish it from the others.  Base class uses "base", new Maya USD class uses "adsk", at the start.